src/transformers/models/conditional_detr/modeling_conditional_detr.py

```diff
@@ -2596,15 +2596,15 @@ def generalized_box_iou(boxes1, boxes2):
     return iou - (area - union) / area
 
 
-# Copied from transformers.models.detr.modeling_detr.center_to_corners_format
-def center_to_corners_format(x):
-    """
-    Converts a PyTorch tensor of bounding boxes of center format (center_x, center_y, width, height) to corners format
-    (x_0, y_0, x_1, y_1).
-    """
-    center_x, center_y, width, height = x.unbind(-1)
-    b = [(center_x - 0.5 * width), (center_y - 0.5 * height), (center_x + 0.5 * width), (center_y + 0.5 * height)]
-    return torch.stack(b, dim=-1)
+# Copied from transformers.image_transforms._center_to_corners_format_torch -> center_to_corners_format
+def center_to_corners_format(bboxes_center):
```

Comment:
* In the case of models, I changed the `# Copied from` statements. Do we want these functions in the model file or to import from `image_tranforms` directly? 
* Maybe `_center_to_corners_format_torch` should be renamed to `center_to_corners_format_torch`? Didn't want to import `center_to_corners_format` to avoid having unnecessary imports in the modeling file.

Reply:
I'd prefer to import `center_to_corners_format`.

Reply:
I've updated so all the models & feature extractors import directly